nativelib/src/main/scala/scala/scalanative/runtime/ExecutionContext.scala

```diff
@@ -25,4 +25,17 @@ object ExecutionContext {
       }
     }
   }
+
+  private[runtime] def loopRunOnce(): Int = {
```

Comment:
I have a concern about duplication of loop() here.

I propose to the discussion that a good way to deal with it is to 
mark/comment this method as experimental, acknowledge the code duplication
with loop(), and say that that duplication
should be eliminated before the method is promoted to supported.

This approach would also explain why the method is, I believe, otherwise undocumented.

I think it makes no sense to disturb the tranquility of the known-to-work loop() 
implementation until the larger picture sorts out.

The classic problem/issue is that loop() wants to be as fast as possible and
does not wanting to be checking all the time if it is one-shot.